### ISSUE TITLE
Unexpected error 105 while listing documents

### ISSUE THREAD

**Issue by jpwenzel** [wneessen/IperBackup#6]
I am seeing an API error code 105 when trying to list the content of my Ipernity account:

``` shell
$ perl IperBackup.pl -c IperBackup.conf -l
2013/02/03 16:15:40 [INFO]: Found 859 documents in total for user account XXX (XXX)
2013/02/03 16:15:40 [INFO]: Retriving list of documents to be fetched...
2013/02/03 16:15:40 [INFO]: Specified timeframe: 1970-01-01 00:00:00 to 2013-02-03 16:15:40
An API call caught an unexpected error: ARRAY(0x7ffe295f6078) (Error Code: 105) at IperBackup/Process.pm line 294
```

My `IperBackup.conf` contains the default values plus the `IPER_API_AUTHTOKEN` and `IPER_USERID` settings.

**Comment by wneessen** [comment]
Which version of Ipernity::API do you have installed on your system?

**Comment by jpwenzel** [comment]
I have just installed `Ipernity::API`, so my CPAN says its the most recent version.

```
cpan[3]> install Ipernity::API            
Ipernity::API is up to date (0.10).
```

**Comment by wneessen** [comment]
Ok, looks like Ipernity changed the API behaviour a little bit. I fixed these issues in Version 0.10 which can be found on Bitbucket now: https://bitbucket.org/wneessen/iperbackup/commits/1aba2114125dff6fab881b476ccf7c0b55f882e1

Just grab the latest versions of IperBackup.pl and IperBackup/Process.pm and replace them on your local installation:
https://bitbucket.org/wneessen/iperbackup/src/1aba2114125d/IperBackup.pl?at=master
https://bitbucket.org/wneessen/iperbackup/src/1aba2114125dff6fab881b476ccf7c0b55f882e1/IperBackup/Process.pm?at=master

That should fix the problem.

**Comment by jpwenzel** [comment]
Problem solved. Thanks for the quick fix.

Are you moving the code base over to Bitbucket?
